frontend/src/components/pages/Scheduling/index.tsx

```diff
@@ -30,23 +31,36 @@ interface UseStepType {
   navigation: NavigationProps | any;
 }
 
-const Scheduling = () => {
-  const [schedulingFormValues, setSchedulingForm] = useForm({
-    id: "",
-    donorId: "",
-    categories: [],
-    size: "",
-    isPickup: false,
-    pickupLocation: "",
-    startTime: new Date().toDateString(),
-    endTime: new Date().toDateString(),
-    status: "",
-    volunteerNeeded: false,
-    frequency: "",
-    notes: "",
-  });
+interface SchedulingProps {
+  schedulingData?: SchedulingFormProps;
```

Comment:
```suggestion
  schedulingData: SchedulingFormProps;
```
this should be mandatory, either the default data or the one being currently edited, right?